test/algolia/integration/search_index_test.rb

```diff
@@ -274,6 +274,9 @@ def test_settings
       @index.set_settings!(settings)
 
       assert_equal @index.get_settings, settings
+
+      # check that the forwardToReplicas parameter is passed correctly
+      assert @index.set_settings!(settings, { forwardToReplicas: true })
```

Comment:
Does this test break _without_ your changes in `lib/algolia/search_index.rb`?

Reply:
indeed! it throws `Algolia::AlgoliaHttpError: Invalid object attributes: forwardToReplicas`